onnxmltools/convert/coreml/shape_calculators/neural_network/Crop.py

```diff
@@ -28,8 +28,8 @@ def calculate_crop_output_shapes(operator):
             output_shape[3] -= params.cropAmounts.borderAmounts[1].startEdgeSize
             output_shape[3] -= params.cropAmounts.borderAmounts[1].endEdgeSize
     elif len(operator.inputs) == 2:
-        output_shape[2] = operator.raw_operator.inputs[1].type.shape[2]
-        output_shape[3] = operator.raw_operator.inputs[1].type.shape[3]
+        output_shape[2] = operator.inputs[1].type.shape[2]
```

Comment:
Better to add this I/O pattern into the doc string (https://github.com/onnx/onnxmltools/pull/119/files#diff-ce871434f7a1bbaccea0f05f280b2c4fR16). For example,
2. [N, C, H, W], [N', C', H', W'] ---> [N, C, H', W']

Reply:
added